install_config/topics/creating_the_bastion_virtual_machine.adoc

```diff
@@ -0,0 +1,156 @@
+////
+Module included in the following assemblies:
+install_config/configuring_rhv.adoc
+////
+[id='creating_the_bastion_virtual_machine_{context}']
+= Creating the bastion virtual machine
+
+Create a bastion virtual machine to install {product-title} on Red Hat Virtualization.
+
+.Procedure
+
+. Log in to the Manager machine by using SSH.
+. Create a temporary bastion installation directory, for example, *_/bastion_installation_*, for the installation files.
+. Create an encrypted *_/bastion_installation/secure_vars.yaml_* file with `ansible-vault`:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-vault create secure_vars.yaml
+----
+
+. Add the following parameter values to the *_secure_vars.yaml_* file:
++
+[source,yml]
+----
+engine_password: <Manager_password> <1>
+bastion_root_password: <bastion_root_password>
+rhsub_user: <Red_Hat_Subscription_Manager_username>
+rhsub_pass: <Red_Hat_Subscription_Manager_password>
+rhsub_pool: <Red_Hat_Subscription_Manager_pool_id>
+root_password: <OpenShift_node_root_password>
+engine_cafile: <Engine_CA_certificate> <2>
```

Comment:
@jan-zmeskal @rgolangh 

Why do we need to specify `engine_cafile` when the location is hard-coded in `create-bastion-machine-playbook.yaml`?